packages/adapter-netlify/README.md

```diff
@@ -17,9 +17,15 @@ import adapter from '@sveltejs/adapter-netlify';
 
 export default {
 	kit: {
+		// default options are shown
 		adapter: adapter({
+			// if true, will create a Netlify Edge Function rather
+			// than using standard Node-based functions
+			edge: false,
+
 			// if true, will split your app into multiple functions
-			// instead of creating a single one for the entire app
+			// instead of creating a single one for the entire app.
+			// if `edge` is true, this option will be disregarded
```

Comment:
```suggestion
			// if `edge` is true, this option cannot be used
```
@Rich-Harris I think the wording needs to be stornger, as we're throwing an error now if they combine them